tools/protoprint/protoprint.py

```diff
@@ -92,7 +93,11 @@ def clang_format(contents):
     Returns:
         clang-formatted string
     """
-    clang_format_path = os.getenv("CLANG_FORMAT", "clang-format")
+    clang_format_path = os.getenv("CLANG_FORMAT", shutil.which("clang-format"))
+    if not clang_format_path:
+        if not os.path.exists("/opt/llvm/bin/clang-format"):
```

Comment:
if you dont use do_ci.sh to trigger this then CLANG_FORMAT is most likely not set (to this) so this worksaround